src/Domain/SizeOfElement.cpp

```diff
@@ -16,6 +16,47 @@
 #include "Utilities/MakeArray.hpp"
 #include "Utilities/StdArrayHelpers.hpp"  // IWYU pragma: keep
 
+namespace {
+template <size_t VolumeDim>
+inline tnsr::I<double, VolumeDim, Frame::Logical> logical_face_center(
+    const size_t logical_dim, const Side& side) noexcept {
+  tnsr::I<double, VolumeDim, Frame::Logical> result{{{0.0}}};
+  result.get(logical_dim) = (side == Side::Lower ? -1.0 : 1.0);
+  return result;
+}
+
+template <size_t VolumeDim>
+inline double distance_between_face_centers(
+    const tnsr::I<double, VolumeDim, Frame::Inertial>& lower_center,
+    const tnsr::I<double, VolumeDim, Frame::Inertial>& upper_center) noexcept {
+  auto center_to_center = make_array<VolumeDim>(0.0);
+  for (size_t inertial_dim = 0; inertial_dim < VolumeDim; ++inertial_dim) {
```

Comment:
replace the for loop with `alg::transform(upper_center, lower_center, center_to_center.begin(), std::minus<>{});`